charts/prometheus-node-exporter/values.yaml

```diff
@@ -40,7 +43,8 @@ updateStrategy:
   rollingUpdate:
     maxUnavailable: 1
 
-resources: {}
+resources:
```

Comment:
please revert

Reply:
Done, sorry I saved and auto-formatted it but thought I'd fully reverted it all.